sgqlc/codegen/schema.py

```diff
@@ -684,6 +685,10 @@ def add_arguments(ap):
                     help=('Include schema descriptions in the generated file '
                           'as docstrings'),
                     default=False)
+    ap.add_argument('--disable-datetime-types', action='store_true',
```

Comment:
likewise, to complement what I said above, we could use some way to specify the types + imports, you could start with a `default_type_imports` copy and then add new types + imports and alternatively prefix the type name with `-` to delete the entry... but I guess in your case you could just replace the import to your desired version

Reply:
Hi, @barbieri! I understood your idea and changed the argument into an argument that can be further developed along the path of including/excluding types. I decided to highlight the argument for excluding default types anyway, because it is easier to describe the exclusion of included types than to describe the inclusion of some external custom types and these paths clearly require separate arguments.